packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2957,6 +3049,122 @@ Generated by [AVA](https://ava.li).
           origin: config␊
           package: ./plugin`
 
+## No --node-path version validation is performed against the plugin engines if the plugins_system_node_version feature flag is set
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags:␊
+        - buildbot_build_plugins_system_node_version␊
+      nodePath: /external/path␊
+      repositoryRoot: packages/build/tests/plugins/fixtures/engines␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      packages/build/tests/plugins/fixtures/engines␊
+    ␊
+    > Config file␊
+      packages/build/tests/plugins/fixtures/engines/netlify.toml␊
+    ␊
+    > Resolved config␊
+      build:␊
+        publish: packages/build/tests/plugins/fixtures/engines␊
+        publishOrigin: default␊
+      functions:␊
+        '*': {}␊
+      plugins:␊
+        - inputs: {}␊
+          origin: config␊
+          package: ./plugin.js␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    > Loading plugins␊
+       - ./plugin.js@1.0.0 from netlify.toml␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. onPreBuild command from ./plugin.js                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    ␊
+    (./plugin.js onPreBuild completed in 1ms)␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build Complete                                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    (Netlify Build completed in 1ms)`
+
+## No --node-path version validation is performed if the plugins_system_node_version feature flag is set
```

Comment:
With the FF set to true, we're just re-runing the test we previously had for this scenario and validating that no error logs are outputted.